druid_test.go

```diff
@@ -19,3 +20,17 @@ func TestSetBaseURLWithSuffix(t *testing.T) {
 	assert.Nil(err, "error should be nil")
 	assert.Equal(d.baseURL, wantBaseURL, "they should not be equal")
 }
+
+func TestNewClientWithSkipVerify(t *testing.T) {
+	assert := assert.New(t)
+
+	var druidOpts []ClientOption
+	druidOpts = append(druidOpts, WithSkipTLSVerify())
+
+	d, err := NewClient("localhost:8082", druidOpts...)
+	assert.Nil(err, "error should be nil")
+	assert.NotNil(d, "client should not be nil")
+	assert.True(d.http.HTTPClient.Transport.(*http.Transport).TLSClientConfig.InsecureSkipVerify, "they should not be equal")
```

Comment:
should not be equal ? maybe "Insecure skip verify should be true" or so instead ?

Reply:
Aah yes, copy paste! :)

Reply:
The statement has been updated